### ISSUE TITLE
Updating status of link not kept when writing INP file

### ISSUE THREAD

**Issue by TimothyVD** [USEPA/WNTR#169]
Hi,

Is it correct that a status change of a valve (e.g. overwriting linkstatus to "Closed") is not kept when using the EpanetSimulator?

If I try to change the linkstatus, it seems successful:
```
link = wn.get_link("LINK_OF_INTEREST")
link.status = 0
wn.get_link("LINK_OF_INTEREST")
```
What yields what I would expect:
```
<Valve 'LINK_OF_INTEREST' from 'A' to 'B', valve_type='TCValve', diameter=0.3, minor_loss=1.5, setting=1.5, status=Closed>
```
However, when running the EpanetSimulator or writing the file, this linkstatus update is gone:
```
<Valve 'LINK_OF_INTEREST' from 'A' to 'B', valve_type='TCValve', diameter=0.3, minor_loss=1.5, setting=1.5, status=Active>
```
I can currently overcome the issue above by manually adding an extra line to the [STATUS] section for that specific valve in the original INP file before opening it with WNTR, which keeps the valve closed during simulation. Is there another way to get to the same desired result?

Kind regards,
Timothy

**Comment by dbhart** [comment]
Hi @TimothyVD 

There is a difference between initial_status and status, and initial_setting and setting. Changing the "status" or "setting" will change the current values on the network (useful for advancing step by step in the WNTRSimulator). Changing the "initial_status" and "initial_setting" will **also** change the current status/setting values. So I think if you use link.initial_status instead, then it will work.

If you have suggestions on how to make these clearer, please let us know.

**Comment by punss** [comment]
Not sure if this is irrelevant now, but I'm having the same issue, where despite changing `initial_status` as well as `status` to Closed, the simulation doesn't run with the updated values.
This is the code I'm using in case additional context is needed.

```
if(el[i-1] == 0):
        wn.get_link(i+21).status = LinkStatus.Closed
        wn.get_link(i+21).intial_status = LinkStatus.Closed
else:
        wn.get_link(i+21).intial_status = LinkStatus.Open
        wn.get_link(i+21).status = LinkStatus.Open
        wn.get_link(i+21).diameter = el[i-1] * 0.0254
```

**Comment by TimothyVD** [comment]
Hi @dbhart,

thanks for the input. That did the trick when writing the file. However, what I perceived as confusing is that the when I changed the "initial_setting", the current setting remains "active". Moreover, for the "initial_setting" I was forced to use "wntr.network.LinkStatus.Closed" to close the valve, while for the "setting" I could just use "0".

@punss : I see you misspelled "initial_status", so probably the reason why you had that issue.

Since the issue seems to be resolved, I will close it.

KR,
Timothy
